README.md

```diff
@@ -235,6 +235,24 @@ If you use our dataset in your academic work, please cite the following paper [[
 - **Description:** Tsudanuma Campus of Chiba Institute of Technology.
 - **License:** TBD
 
+### UTsukuba 2022, ROBOKEN, Map Data
+
+- **Short Name:** map_utsukuba22_roboken
+- **Provider (Team):** ROBOKEN
+- **Location:** Tsukuba campus 2022
+- **Type:** Map
+- **File:** [map_utsukuba22_roboken.pcd](https://drive.google.com/file/d/1IryFbyc9VYZTTtD53FgluXEaUYqctv0h/view?usp=sharing)
+- **Size:** 41 MB
+- **Format:** pcd
+- **Number of Points:** 2,875,954
+- **Point Type:**
+  - **XYZ:** Yes
+  - **Intensity:** Yes
+  - **Color:** No
+  - **Normal:** Yes
+- **SLAM Method:** LIO-SAM
+- **Description:** Tsukuba Campus of University of Tsukuba.
+- **License:** TBD
```

Comment:
- Set your license. "CC BY-NC-SA 4.0" is recommended.

Reply:
Added the license